idseq_dag/util/s3.py

```diff
@@ -146,58 +163,33 @@ def fetch_from_s3(src,  # pylint: disable=dangerous-default-value
     untar = auto_untar and dst.lower().endswith(".tar")
     if untar:
         dst = dst[:-4]  # Remove .tar
-    abspath = os.path.abspath(dst)
 
+    destdir = os.path.dirname(dst)
+    tmp_destdir = os.path.join(destdir, "tmp_downloads")
```

Comment:
We use `/foo/bar/tmp_downloads/dst` for staging instead of `/foo/bar/dst.download` because we need to handle correctly the case when a TAR reference will be autoexpanded.